### PR TITLE
Fix problems with a blank HOST field (Doesn't work either)

### PR DESCRIPTION
**Pull request by ghost**
Previously sed didn't get a match if the HOST field was empty.

### CODE REVIEW

File: ddns-start [hunk 1]
```diff
@@ -3,11 +3,12 @@
 APIKEY="apikey"
 DOMAIN="domain"
 HOST="host"
+FULLDOMAIN=$HOST.$DOMAIN
```
Comment (alphabt) [inline ddns-start:6]: If $HOST is blank, will $FULLDOMAIN start with an unnecessary dot like ".example.com"?

**Review by alphabt — COMMENTED** [review]
Thanks for the fix. I appreciate your updated approach on defining a FQDN.

File: ddns-start (hunk 1 above)
Comment (ghost) [reply ddns-start:6]: Fuuck. Yeah, it will. Gotta refine this.

Comment (ghost) [reply ddns-start:6]: I literally made one version which worked only with plain domains and another one which was a copy of the original. Whatever, now it should be set.

**Pull request closed without merging**